src/StatusQ/Core/StatusIcon.qml

```diff
@@ -19,7 +19,7 @@ Image {
     }
 
     ColorOverlay {
-        visible: statusIcon.color !== undefined
+        visible: !!statusIcon.color
```

Comment:
i think this maybe more pretty:
`layer.enabled: !!statusIcon.color
  layer.effect: ColorOverlay {
      color: statusIcon.color
  }`

Reply:
Oh nice, yes, will do that